### PR TITLE
Add policy document

### PR DESCRIPTION
**Pull request by briandowns**
#### Linked Issues ####

#327

### CODE REVIEW

**Review by galal-hussein — APPROVED** [review]
LGTM

**Review by dweomer — APPROVED** [review]
The network policy yaml have `managedFields` which I assume come from `kubectl get -o yaml ...` which is fine but these could be trimmed to make the yaml as presented more concise.

**Comment by briandowns** [comment]
> The network policy yaml have `managedFields` which I assume come from `kubectl get -o yaml ...` which is fine but these could be trimmed to make the yaml as presented more concise.

Great catch. I appreciate it. I thought I got them all.

File: docs/policies.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+kind: NetworkPolicy
+metadata:
+  managedFields:
+  - apiVersion: networking.k8s.io/v1
+    fieldsType: FieldsV1
+    fieldsV1:
+      f:spec:
+        f:ingress: {}
+        f:policyTypes: {}
+    manager: rke2
```
Comment (dweomer) [inline docs/policies.md:141]: This can be removed I think.

File: docs/policies.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+metadata:
+  managedFields:
+  - apiVersion: networking.k8s.io/v1
+    fieldsV1:
+      f:spec:
+        f:ingress: {}
+        f:podSelector:
+          f:matchLabels:
+        f:policyTypes: {}
+    manager: rke2
```
Comment (dweomer) [inline docs/policies.md:167]: This can also be removed.

**Pull request merged**